# This service is used to update an existing target on the interoperability
# server.

# Target ID to update.
uint64 id

# Target with updated characteristics.
Target target

---

# Whether the request was successful.
bool success
